soc/x86/elkhart_lake/Kconfig.soc

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2018-2020 Intel Corporation Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_ELKHART_LAKE
+	bool "Intel Elkhart Lake Soc"
+	select X86
+	select CPU_ATOM
+	select HAS_I2C_DW if I2C
+	select PCIE
+	select PCIE_MSI
+	select DYNAMIC_INTERRUPTS
```

Comment:
at some point I think we should move DYNAMIC_INTERRUPTS and X86_MMU selections to CONFIG_X86_64 since the arch itself requires it. no need to do here.